doc/include/api_stubs.inc

```diff
@@ -1,122 +1,96 @@
+[option --with-[var name]-include].
 
 [list_end]
 
 
-[para] Second, declaration and export of a stubs table, i.e. API, for
-the current package, [var foo]:
+[para] Declaration and export of a stubs table, i.e. API, for
+the [vset critcl_script]:
 
 [list_begin definitions]
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::api] [method function] [arg resulttype] [arg name] [arg arguments]]
 
-This method declares that the function [arg name] is in the public API
-of the package, and its signature (type of the result, number, names
-and types of its arguments).
-
-Using this method automatically causes critcl to generate both the
-code for a stubs table in the package, the headers needed by packages
-using this API, and a [file .decls] file containing the stubs table
-implied by the exports, usable by [cmd {critcl::api import}].
-
-[para] [arg arguments] is a list of C types and associated argument
-names. Like a dictionary, except that keys (argument names) and values
-(argument types) are swapped. The [arg resulttype] is a C type as well.
+Adds to the public API of the [vset critcl_script] the signature the
+for the function named name [arg name] having the signature specified by
+[arg arguments] and [arg resulttype].  Code is generated for a [file .decls]
+file, the corresponding public headers, and a stubs table usable by
+[cmd {critcl::api import}].  [para] [arg arguments] is a multidict where each
+key is an argument type and its value is the argument name, and
+[arg resulttype] is a C type.
 
 [comment ---------------------------------------------------------------------]
-[call [cmd ::critcl::api] [method header] [opt [arg pattern]...]]
-
-This method notifies critcl of companion header files which have to be
-exported together with the generated stubs headers.
+[call [cmd ::critcl::api] [method header] [opt [arg {glob pattern}]...]]
 
-[para] All arguments are interpreted as glob pattern and the matching
-files are copied into the directory containing the generated headers
-well. As an importing package uses only [file [var foo]Decls.h] to
-access the API this generated header will contain the necessary
-[term {#include}] directives to make these companion header files and their
-declarations available too.
+Each file matching a [arg {glob pattern}] is copied into the directory
+containing the generated headers, and an [term {#include}] directive for it is
+added to [file Decls.h] for the [vset critcl_script].
 
-Patterns matching no file or non-existing files cause the command to
-throw an error.
+Returns an error if a [arg {glob pattern}] matches nothing.
 
-[para] [emph Note] that patterns which are not beginning with an
-absolute path are interpreted relative to the directory containing the
-current [file .critcl] file.
+[para] A pattern for a relative pathname is resolved relative to the
+directory containing the [vset critcl_script].
```

Comment:
`[vset relative_pattern]` ?

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).